test/common.coffee

```diff
@@ -23,6 +23,17 @@ describe 'Common functions', () ->
       assert.equal defaultCam.path, '/onvif/device_service'
       done()
 
+  describe 'autoconnect disabled', () ->
```

Comment:
Please add another test where the `autoconnect` option isn't passed in and it **does**  connect

Reply:
ok, done